.github/workflows/load-tests.yml

```diff
@@ -4,34 +4,43 @@ run-name: Load tests by @${{ github.actor }}
 on: workflow_dispatch
 
 jobs:
-  perf-tests:
+  k6:
```

Comment:
I probably prefer perf-tests or load-tests. I am not familiar with k6 but if it says load-test or perf-test I would have a general idea of what's supposed to happen in the job.

Reply:
See other response